Documentation/development-flow.md

```diff
@@ -35,6 +35,14 @@ cd $GOPATH/src/github.com/rook
 git clone git@github.com:<user>/rook.git
 
 cd rook
+
+# Installs Go dependancies
```

Comment:
The build instructions are in INSTALL.md in the root of the repo. Maybe you could include a link to it here instead. But it wouldn't hurt to mention the `make` command here at the root of the repo, no need to go to the `cmd/rook` folder. And if you run `make` i believe it should take care of the `dep ensure`.

Reply:
done